warehouse/workflows/extraction/__init__.py

```diff
@@ -16,9 +17,8 @@ def __call__(self):
 
     @staticmethod
     def add_args(parser):
-        name = 'extract'
         p = parser.add_parser(
-            name=name,
```

Comment:
Fascinating. . . totally mysterious.

Reply:
This is a mistake, it should be NAME. The reason I broke it up like that, is because when you're dynamically loading the module, you have access to the module level variables, but you dont have access to the class variable until you instantiate it, so i moved the name up to the module level constant NAME.